views/includes/page-templates/beneficial-owner-individual.html

```diff
@@ -45,6 +45,8 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
       {% set sanctionsListText = "Are they on the sanctions list?" %}
       {% set sanctionsListHintText = "This means that they are subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
+      {% set sanctionsListNoText = "No, they are not on the Sanctions List" %}
```

Comment:
The `sanctionsListText` should be `Are they on the UK Sanctions List?` as for AC in your story

Reply:
Updated this to the precise text.